test/sql/index/art/constraints/test_art_concurrent_upserts.test

```diff
@@ -0,0 +1,203 @@
+# name: test/sql/index/art/constraints/test_art_concurrent_upserts.test
+# description: Test UPSERT in the main connection with concurrent upserts in other connections.
+# group: [constraints]
+
+statement ok
+PRAGMA enable_verification
+
+statement ok
+CREATE TABLE tbl (id INT PRIMARY KEY, payload VARCHAR);
```

Comment:
Can we remove this test with `VARCHAR` since it doesn't test anything new (this already used to work)?